### PR TITLE
expose max client session per auth name for event grid.

### PR DESCRIPTION
**Pull request by morvencao**
### What this PR does

This PR attempts to expose the max client session per auth name parameter for event grid MQTT broker.

Jira: 
Link to demo recording: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Deployment: The deployment process was considered and addressed if required
- [ ] Testing: New code requires new unit tests.
- [ ] Documentation: Is the documentation updated? Either in the doc located in focus area, in the README or in the code itself.
- [ ] Customers: Is this change affecting customers? Is the release plan considered?

### CODE REVIEW

**Review by mjlshen — APPROVED** [review]

**Review by geoberle — DISMISSED** [review]

File: dev-infrastructure/configurations/svc-cluster.bicepparam [hunk 1]
```diff
@@ -11,6 +11,7 @@ param disableLocalAuth = false
 param deployFrontendCosmos = false
 param deployMaestroInfra = false
 param maestroNamespace = 'maestro'
+param maxClientSessionsPerAuthName = 1
```
Comment (machi1990) [inline dev-infrastructure/configurations/svc-cluster.bicepparam:14]: Should the default be 2 to cover for pod rollout?

Comment (morvencao) [reply dev-infrastructure/configurations/svc-cluster.bicepparam]: updated default value to 2.

**Comment by morvencao** [comment]
rebased the code.

**Review by petrkotas — APPROVED** [review]
LGTM

**Pull request merged**